astropy/cosmology/flrw/conftest.py

```diff
@@ -0,0 +1,6 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Do we need this backwards compatibility?

Reply:
Oops. That was meant to be in ``flrw/tests/``. Thanks!